playbooks/vars/all.yml

```diff
@@ -27,3 +27,15 @@ default_gating_overrides:
   # too low for the liberty->mitaka upgrade job where more space is used by
   # additional packages, venvs, logs, etc.
   percent_used_critical_threshold: 95
+
+ansible_ssh_extra_args: >
```

Comment:
These need to be in ``playbooks/inventory/group_vars/all.yml`` - this vars file is only used for some RPC-O configuraiton playbooks and does not apply to the single use slave setup.